test/types/string/methods/testDedent.chpl

```diff
@@ -0,0 +1,89 @@
+/* Temporary dummy method to make sure I can compile this test */
+proc string.dedent(columns=0, ignoreFirst=true) { return this; }
+
+var testStrings = [
+
+  // [0] Common use-case
+  """a
+     b
+     c""".dedent(),
+
+  // [1] Newlines before/after
+  """
+  a
+  b
+  c
```

Comment:
This case seems like it would be more interesting if each line started with a few spaces?  Or else, I'd clone it and create a version that was like that as well?

Reply:
I included the less interesting version here because it's a common use-case. I can include a more interesting one as well.